src/main/webapp/app/lecture/overview/course-lectures/attachment-video-unit/attachment-video-unit.component.spec.ts

```
@@ -414,4 +425,152 @@ describe('AttachmentVideoUnitComponent', () => {
+
+            expect(component.playlistUrl()).toBe(playlist);
+            expect(component.pdfUrl()).toBe(mockUrl);
+        });
+
+        it('ngOnDestroy: cleanup', async () => {
+            const mockUrl = 'blob:http://localhost/test-pdf';
+            component.pdfUrl.set(mockUrl);
+            component.lectureUnit().attachment!.link = '/path/to/file/test.pdf';
+
+            // Clean up any pending requests first
+            httpMock.match(() => true).forEach((req) => req.flush(new Blob()));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)